OpenSim/Actuators/Thelen2003Muscle.h

```diff
@@ -151,12 +138,11 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Thelen2003Muscle, ActivationFiberLengthMuscle);
     OpenSim_DECLARE_PROPERTY(Flen, double,
         "maximum normalized lengthening force");
 
-    //OpenSim_DECLARE_PROPERTY(activation_minimum_value, double,
-    //    "minimum activation value permitted");
-    
-    OpenSim_DECLARE_PROPERTY(fv_linear_extrap_threshold, double,
+    OpenSim_DECLARE_PROPERTY(FvExtrapolationThreshold, double,
```

Comment:
Oh...I was going to suggest leaving them as is. Changing the property names might create a headache for reading old XML files.